modules/os2forms_nemid/src/Element/MitidChildrenSelectAjaxBehaviour.php

```diff
@@ -52,6 +52,10 @@ public static function mitidChildrenSelectAjax(array &$form, FormStateInterface
       $webform = $webformSubmission->getWebform();
       $elementsFlattened = $webform->getElementsInitializedAndFlattened();
 
+      if (!$cprLookupResult->isSuccessful()) {
+        return $response;
+      }
+
```

Comment:
Any specific reasons for not making this check right after the call to `$cprPlugin->lookup` (i.e. in line 47)?